encord/storage.py

```diff
@@ -1056,6 +1097,8 @@ def _get_content_type(self, file_path: Union[Path, str], item_type: StorageItemT
         # TODO QUESTION: Not entirely sure if this is right.
```

Comment:
COuld u help me remove this comment? hehe

Reply:
sure